reference/behavior_test/spin.rst

```diff
@@ -0,0 +1,85 @@
+Using Spin In Behat
+===================
+
+As our Behat tests are pretty slow we had problem for waiting that some Element appears in pages. Sometimes an Element
+is not found because the page don't finished to load. In this case, waiting two or three seconds more could help to
+find elements.
+
+.. note::
+
+    Learn about spin from `Behat documentation <http://docs.behat.org/en/v2.5/cookbook/using_spin_functions.html>`_.
+
+Use the SpinCapableTrait
+------------------------
+
+In Akeneo PIM we use spin as a Trait. To be able to use the spin method you only have to add the Trait in your decorator
+or your context as follows:
+
+.. code-block:: php
+
+    use Context\Spin\SpinCapableTrait;
+    use Pim\Behat\Decorator\ElementDecorator;
+
+    MyClassDecorator extends ElementDecorator {
+        use SpinCapableTrait;
+    }
+
+Spin Signature
+--------------
+
+In the method signature you can see two arguments.
+
+.. code-block:: php
+
+    // Context\Spin\SpinCapableTrait
+
+    public function spin($callable, $message) {}
+
+The first one is a callable function. This function is where you search for your Element. If this callable returns
+false, null or throws an exception, the spin will retry every second the callable function until it returns a value or
+the loop limit is reached.
+
+The second one is the message that will be rendered in the console in case the loop limit is reached and the callable
```

Comment:
will be written [...] in case the loop limit is reached. (end of sentence is paraphrasing, if loop limit is reached it can only be a consequence of the callable function failing each time, am I correct?)